src/components/Form/Field.js

```diff
@@ -41,12 +41,22 @@ const styles = {
     borderRadius: 0,
     backgroundColor: 'transparent'
   }),
+  codeField: css({
+    fontFamily: 'Courier, Courier New, monospace',
+    fontSize: 14
+  }),
   labelText: css({
     position: 'absolute',
     left: X_PADDING,
     top: LINE_HEIGHT + Y_PADDING,
     transition: 'top 200ms, font-size 200ms'
   }),
+  labelTextCode: css({
+    position: 'absolute',
+    left: X_PADDING,
+    top: LINE_HEIGHT + Y_PADDING,
+    transition: 'top 200ms, font-size 200ms'
+  }),
```

Comment:
isn't this already deployed?

Reply:
good catch, i should delete it. I had to overwrite it in publikator itself.